src/lib/OpenEXR/ImfStandardAttributes.h

```diff
@@ -232,6 +530,51 @@ IMF_STD_ATTRIBUTE_DEF (timeCode, TimeCode, TimeCode)
 
 IMF_STD_ATTRIBUTE_DEF (framesPerSecond, FramesPerSecond, Rational)
 
+//
+// timecodeRate -- the nominal playback rate associated with
+// the image sequence to which the image belongs, specified in
+// timecodes per second.
+//
+// The integer timecodeRate attribute is a rounding of the
+// framesPerSecond attribute.
+//
+// The rounding is atypical: in the special case of the nominal
+// playback rate corresponding exactly to the midpoint of two
+// adjacent integer values, timecodeRate is defined to always
+// be the rounding up to the next larlgest integer.
+//
+// This stands in contrast to the default IEEE 754 rounding for
+// decimal arithmetic in which such a midpoint value would always
+// be rounded to the nearest even integer.
+//
+
+IMF_STD_ATTRIBUTE_DEF (timecodeRate, TimecodeRate, int)
+
+//
+// imageCounter -- an image number
+//
+// For a sequence of images, the image number increases 
+// when the images are accessed in the intended play order. 
+// imageCounter can be used to order frames when more standard
+// ordering systems are inapplicable, including but not limited
+// to uniquely identifying frames of high-speed photography that
+// would have identical time codes, ordering sequences of frames
+// where some frames may have been captured and discarded due to
+// real-time constraints, or ordering frames in a sequence that
+// is intermittently accumulated from devices such as security
+// cameras triggered by motion in an environment.
+//
```

Comment:
Would we also want to say that in the case of scanned motion picture film, imageCounter corresponds to the embossed edge codes on the film?